Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -312,36 +185,27 @@ open class LottieAnimationView: LottieAnimationViewBase {
     loopMode: LottieLoopMode? = nil,
     completion: LottieCompletionBlock? = nil)
   {
-    guard let from = animation?.markerMap?[marker] else {
-      return
-    }
-
-    play(
-      fromFrame: from.frameTime,
-      toFrame: from.frameTime + from.durationFrameTime,
-      loopMode: loopMode,
-      completion: completion)
+    self.lottieAnimationLayer.play(marker: marker, loopMode: loopMode, completion: completion)
   }
 
   /// Stops the animation and resets the view to its start frame.
   ///
   /// The completion closure will be called with `false`
   open func stop() {
-    removeCurrentAnimation()
-    currentFrame = 0
+    self.lottieAnimationLayer.stop()
   }
 
   /// Pauses the animation in its current state.
   ///
   /// The completion closure will be called with `false`
   open func pause() {
-    removeCurrentAnimation()
+    self.lottieAnimationLayer.pause()
   }
 
   // MARK: Public
 
-  /// The configuration that this `LottieAnimationView` uses when playing its animation
-  public let configuration: LottieConfiguration
+  // The backing CALayer for this animation view.
+  public var lottieAnimationLayer: LottieAnimationLayer;
```

Comment:
Can this be `private` and immutable? 

```suggestion
  private let lottieAnimationLayer: LottieAnimationLayer
```

Reply:
I'd prefer to not increase the public API surface area unless really necessary